src/main/java/com/stevedutch/assignment14/repository/MessageRepository.java

```diff
@@ -1,28 +1,11 @@
 package com.stevedutch.assignment14.repository;
 
-import java.util.ArrayList;
-import java.util.List;
-
+import com.stevedutch.assignment14.domain.Message;
 import org.springframework.data.jpa.repository.JpaRepository;
-import org.springframework.data.jpa.repository.Query;
 
-import com.stevedutch.assignment14.domain.Message;
+import java.util.List;
 
 public interface MessageRepository extends JpaRepository <Message, Long> {
 
-//	 XXX To use native SQL queries with Spring Data JPA, you need to add the nativeQuery parameter 
```

Comment:
- there's no need to make the queries as native queries because then they will work only on that given database, we can use the Spring JPA style and create a method like `List<Message> findTop10ByOrderByMessageIdDesc()` to achieve the same result
- to keep your code clean, it's recommended to delete all unused methods from your classes
- no need to declare interface methods as public since actually they must be public to allow for overriding when you implement the interface and are public by default